Assets/MixedRealityToolkit.Examples/Demos/UX/Interactables/Scenes/InteractablesExamples.unity

```diff
@@ -38,7 +38,7 @@ RenderSettings:
   m_ReflectionIntensity: 1
   m_CustomReflection: {fileID: 0}
   m_Sun: {fileID: 0}
-  m_IndirectSpecularColor: {r: 0.44657898, g: 0.4964133, b: 0.5748178, a: 1}
+  m_IndirectSpecularColor: {r: 0.44657844, g: 0.49641222, b: 0.57481694, a: 1}
```

Comment:
Thanks for all the great comments and questions, there are a lot of great points here. I'll start implementing the actionable feedback and try to answer what I can.

The main purpose of this work was to try to get more feedback options into the Interactable Theme system, like grip (grab) and physical press. The PhysicalPressEventRouter and PressableButton can use Interactable themes for simple feedback, if what we have now is sufficient, then we can reduce the scope of this PR. I was contemplating adding the ability to turn off all interaction handling, so that other components could handle that.

The grip/grab functionality is purely visual - to drive themes, it doesn't trigger an OnClick event. Though the event/receiver system is built to look for state patterns and trigger events, like the InteractableGripReceiver. It provides a Unity event for OnGrip and OnRelease (I should change the current naming).

In order to figure out if a grip is happening, I needed to add a bunch of code though, so I like the idea of abstracting some of this stuff out, especially to reduce the duplicated code. I'm also really interested in anything we can do to abstract pointer, hand and controller data and make that much easier to pass around and decipher intent. There was an effort to create a base button class that we could extend, the idea was to have Interactable extend that class and focus on the themes and feedback.

I wasn't thinking about adding pressable button functionality to Interactable as much as trying to make it easy for a button that already works with air tap and controller interaction, to also understand touch. It's a subtle, but the outcome was the same, duplicated code.

Reply:
> In order to figure out if a grip is happening, I needed to add a bunch of code though,

I wan to understand this better. Could you point me to the code in your PR that you are using to determine if Grip is happening?

Reply:
Updated the info panel and simplified how grab is being detected. Right now we are detecting NearInteractionGrabbable interaction.